docker/worker/worker.py

```diff
@@ -389,6 +390,28 @@ def _source_update(self, message):
         return
 
       repo = None
+    elif source_repo.type == osv.SourceRepositoryType.REST_ENDPOINT:
+      vulnerabilities = []
+      if source_repo.link[-1] != '/':
```

Comment:
same comment here: let's just enforce and make the assumption that these links must always end in /.